tutorials/super_blink/super_blink.c

```diff
@@ -0,0 +1,51 @@
+#include "blink_config.h"
+
+#include <avr/interrupt.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define HW_TIMER_Hz     (30)
+#define SW_TIMER_SCALER (6)
+
+static volatile bool run_blink_led0 = true;
+static volatile bool run_blink_led1 = true;
```

Comment:
Since this is not being changed in an ISR (hardware interrupt), it doesn't need to be designated as volatile